src/summary/ScatterPlotVisualizer.jsx

```diff
@@ -13,16 +13,25 @@ class ScatterPlotVisualizer extends Component {
 
         this.updateState = true;
         this.state = {
-            chartWidth: 600,
-            chartHeight: 400,
-        }
+            chartWidth: "600px",
+            chartHeight: "600px",
+        } 
     }
 
     componentDidMount() {
         const { patient, condition, conditionSection } = this.props;
         this.renderScatterPlot(patient, condition, conditionSection);
     }
 
+     componentDidUpdate = () => {
```

Comment:
Do we need this function still? I don't think we want this graph to drop to 30px whenever the data for the visualization changes... Is this possibly an artifact of a copy-paste job?

Reply:
I removed the function since it's not being used.